debugger/src/192/main/kotlin/org/rust/debugger/runconfig/RsDebugProcessConfigurator.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.debugger.runconfig
+
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.openapi.util.text.StringUtil
+import com.jetbrains.cidr.execution.debugger.CidrDebugProcess
+import com.jetbrains.cidr.execution.debugger.CidrDebugProcessConfigurator
+import com.jetbrains.cidr.execution.debugger.backend.DebuggerCommandException
+import com.jetbrains.cidr.execution.debugger.backend.DebuggerDriver
+import com.jetbrains.cidr.execution.debugger.backend.gdb.GDBDriver
+import com.jetbrains.cidr.execution.debugger.backend.lldb.LLDBDriver
+import org.rust.cargo.project.model.cargoProjects
+import org.rust.cargo.project.settings.toolchain
+import org.rust.cargo.runconfig.command.workingDirectory
+import org.rust.debugger.*
+import org.rust.debugger.settings.RsDebuggerSettings
+import org.rust.ide.notifications.showBalloon
+import java.nio.file.InvalidPathException
+
+class RsDebugProcessConfigurator : CidrDebugProcessConfigurator {
+    override fun configure(process: CidrDebugProcess) {
+        val project = process.project
+        val toolchain = project.toolchain
+        val cargoProject = project.cargoProjects.allProjects.firstOrNull()
```

Comment:
If we have more than one cargo project we may get wrong one here.
Let's better use `CargoCommandConfiguration#findCargoProject` here.
If you need to pass additional params to do it, you can pass them via `RsDebugRunParameters` and extract them via `CidrDebugProcess#getRunParameters`